lightstep/propagation.py

```diff
@@ -4,6 +4,10 @@
 
 from __future__ import absolute_import
 
+# The B3 format represents SpanContexts in B3 format.
+# https://github.com/openzipkin/b3-propagation
+B3 = "b3"
```

Comment:
Do we really need it here?

Reply:
Well, it can be placed somewhere else if you prefer, of course. I just added it there since that module was holding the register identifier for the other propagator in the package, `LightStepBinaryPropagator`.